<?xml version="1.0"?>
<launch>
    <arg name="config_file" default="$(find aruco_proxy_pose_estimation)/config/apriltag_grid_detector_config.yaml"/> <!--provide path the the detector config file here-->
    <node pkg="aruco_proxy_pose_estimation" type="apriltag_grid_detection.py" name="apriltag_grid_detector" output="screen">
         <rosparam command="load" file="$(arg config_file)"/>
    </node>
</launch>